build.gradle

```diff
@@ -1,25 +1,23 @@
 buildscript {
 	repositories {
-		mavenCentral()
-		maven {
-			name = "forge"
-			url = "http://files.minecraftforge.net/maven"
-		}
-		maven {
+        jcenter()
+        maven {
+            name = "forge"
+            url = "http://files.minecraftforge.net/maven"
+        }
+				maven {
 			name = "sonatype"
 			url = "https://oss.sonatype.org/content/repositories/snapshots/"
 		}
-	}
+    }
 	dependencies {
-		classpath 'net.minecraftforge.gradle:ForgeGradle:1.2-SNAPSHOT'
-		// https://github.com/ajoberstar/gradle-git
-		classpath 'org.ajoberstar:gradle-git:0.8.0'
+		classpath 'net.minecraftforge.gradle:ForgeGradle:2.1-SNAPSHOT'
 	}
 }
 
-apply plugin: 'forge'
-apply plugin: 'maven'
-apply plugin: 'java'
+plugins {
+    id "net.minecraftforge.gradle.forge" version "2.0.2"
```

Comment:
You do not need this plugin actually, as you have already used it in buildscript. (`classpath 'net.minecraftforge.gradle:ForgeGradle:2.1-SNAPSHOT'`)